src/main/java/org/folio/models/TemplateProcessingRequest.java

```diff
@@ -0,0 +1,156 @@
+package org.folio.models;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class TemplateProcessingRequest {
+  @JsonProperty
+  private UUID templateId;
+  @JsonProperty
+  private String lang;
+  @JsonProperty
+  private String outputFormat;
+  @JsonProperty
+  private Context context;
+
+  public UUID getTemplateId() {
+    return templateId;
+  }
+
+  public String getLang() {
+    return lang;
+  }
+
+  public String getOutputFormat() {
+    return outputFormat;
+  }
+
+  public Context getContext() {
+    return context;
+  }
+
+  public TemplateProcessingRequest withTemplateId(UUID templateId) {
+    this.templateId = templateId;
+    return this;
+  }
+
+  public TemplateProcessingRequest withLang(String lang) {
+    this.lang = lang;
+    return this;
+  }
+
+  public TemplateProcessingRequest withOutputFormat(String outputFormat) {
+    this.outputFormat = outputFormat;
+    return this;
+  }
+
+  public TemplateProcessingRequest withContext(Context context) {
+    this.context = context;
+    return this;
+  }
+
+  public static class Context {
+    @JsonProperty
+    private RoutingList routingList;
+    @JsonProperty
+    private List<User> users;
+
+    public RoutingList getRoutingList() {
+      return routingList;
+    }
+
+    public List<User> getUsers() {
+      return users;
+    }
+
+    public Context withRoutingList(RoutingList routingList) {
+      this.routingList = routingList;
+      return this;
+    }
+
+    public Context withUsers(List<User> users) {
+      this.users = users;
+      return this;
+    }
+  }
+
+  public static class RoutingList {
```

Comment:
can we use RoutingList from acq-models, that we already merged?

Reply:
used, thanks